### PR TITLE
Make drawer demo header not scroll away

### LINKED ISSUE #13335: UserAccountsDrawerHeader should be a persistent sliver header or not be in a ListView
In the drawer demo, the accounts header is part of the list and can scroll down. 

flutter_01

### PR DESCRIPTION
**Pull request by xster**
Fixes https://github.com/flutter/flutter/issues/13335

Also update gallery xcode project with new plugin dependencies

### CODE REVIEW

File: examples/flutter_gallery/ios/Runner.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -508,4 +514,4 @@
 /* End XCConfigurationList section */
 	};
 	rootObject = 97C146E61CF9000F007C117D /* Project object */;
-} 
\ No newline at end of file
+}
```
Comment (Hixie) [inline examples/flutter_gallery/ios/Runner.xcodeproj/project.pbxproj]: ?

**Comment by Hixie** [comment]
test?
LGTM

File: examples/flutter_gallery/ios/Runner.xcodeproj/project.pbxproj (hunk 1 above)
Comment (xster) [reply examples/flutter_gallery/ios/Runner.xcodeproj/project.pbxproj]: Xcode just auto-adds final newlines

**Comment by xster** [comment]
Added test and fixed account header overflow text. Had to refactor a bit.

File: examples/flutter_gallery/ios/Runner.xcodeproj/project.pbxproj (hunk 1 above)
Comment (Hixie) [reply examples/flutter_gallery/ios/Runner.xcodeproj/project.pbxproj]: I didn't mean the newline, I meant all the changes to this file. They seem unrelated to the patch.

Comment (xster) [reply examples/flutter_gallery/ios/Runner.xcodeproj/project.pbxproj]: Oh I see. Ya, it's somewhat unrelated. Leftovers from https://github.com/flutter/flutter/pull/13195 which added new plugins. We'll just get these diffs if anyone flutter runs on iOS until we check them in.

**Pull request merged**